docs/src/explanations/interfaces-and-connections.md

```diff
@@ -67,8 +68,34 @@ class CrossbarIo(n: Int) extends Bundle {
 where Vec takes a size as the first argument and a block returning a port as the second argument.
 
 ## Bulk Connections
+Once we have a defined Interface, we can connect to it via a [`MonoConnect`](../../core/src/main/scala/chisel3/internal/MonoConnect.scala) operator (`:=`) or [`BiConnect`](../../core/src/main/scala/chisel3/internal/BiConnect.scala) operator (`<>`).
```

Comment:
These relative links won't work on the website. Usually we link to the API docs here. For now let's just remove the links
```suggestion
Once we have a defined Interface, we can connect to it via a [`MonoConnect`](https://www.chisel-lang.org/api/latest/chisel3/Data.html#:=) operator (`:=`) or [`BiConnect`](https://www.chisel-lang.org/api/latest/chisel3/Data.html#%3C%3E) operator (`<>`).
```